package.json

```diff
@@ -6,15 +6,13 @@
   "private": true,
   "scripts": {
     "preinstall": "git config --global url.\"https://\".insteadOf ssh://",
-    "install-pkg": "yarn install --link-duplicates",
     "lint": "yarn lerna run lint && yarn run --silent lint:do-not-merge",
     "lint:do-not-merge": "! git grep -E 'DO[ _]*NOT[ _]*MERGE'",
-    "prettify": "yarn run prettier --config .prettierrc.js --write '**/*.+(ts|tsx|js|jsx|sol|java)'",
-    "prettify:diff": "yarn run prettier --config .prettierrc.js --list-different '**/*.+(ts|tsx|js|jsx|sol|java)'",
+    "prettify": "yarn run prettier --config .prettierrc.js --write '**/*.+(ts|js|sol)'",
+    "prettify:diff": "yarn run prettier --config .prettierrc.js --list-different '**/*.+(ts|js|sol)'",
     "reset": "yarn reset-modules && yarn reset-cache",
-    "reset-cache": "yarn reset-yarn && yarn reset-rn",
+    "reset-cache": "yarn reset-yarn",
     "reset-modules": "rm -rf node_modules/ packages/*/node_modules",
-    "reset-rn": "watchman watch-del-all; rm -rf $TMPDIR/metro-cache-*; rm -rf $TMPDIR/haste-map-*; rm -rf $TMPDIR/metro-symbolicate*",
```

Comment:
This command is related to reseting react-native and should have been remove long ago